source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,217 @@ This area will contain the Migration steps to follow for upgrading your store to
+</blockquote>
+
+### Compatibility with Magento modules 1.x+
+
+Feature flags were implemented in the core to support 3-years old Magento modules. We've dropped this support and recommend that you ensure to have a Magento module version greater than 1.0.0!
+
+### Sharp and libvips minor update
+
+[`sharp`](https://www.npmjs.com/package/sharp) has been updated from 0.23 to 0.25, which requires libvips v8.9.1 (instead of v8.8.1).
+You may have to rebuild it by removing your `node_modules` folder or run `npm rebuild`.
+
+Further details available on https://sharp.pixelplumbing.com/changelog.
+
+### `date-fns` isn't a Front-Commerce dependency anymore
+
+[`date-fns`](https://www.npmjs.com/package/date-fns) has been removed from dependencies in favor of a native `winston` plugin that timestamps log entries, and custom functions elsewhere.
+
+It allows developer to choose the version of `date-fns` they prefer and reduce the bundle size. Please ensure you are not using `date-fns` in your application, or install it in your project. The version in Front-Commerce was 1.30.1, and the latest available is 2.14.0.
+
+For backwards compatibility, you can run:
+```shell
+npm install --save date-fns@1.30.1
+```
+
+> **Logging format update:** a timestamp is now automatically added to log messages as the `timestamp` field of each entry (in GMT time). `date` fields were removed. You can update your custom logs to do so too.
+
+See [this commit](https://gitlab.com/front-commerce/front-commerce/-/commit/bf9f9ace04bc24a7391fe36c267dd94857663db0) and [this one](https://gitlab.com/front-commerce/front-commerce/-/commit/60c31720fe3f113961555c493417b8c8bd45509b) for an overview of those changes.
+
+### Make linting pass again
+
+Dependencies were updated, a first step would be to automatically update as much as your codebase as possible to ensure it matches the latest conventions and good practices.
+You can run:
+
+```shell
+npm run lint -- --fix
+```
+
+Below are the main updates to be aware of.
+
+#### Prettier updated to 2.x
+
+We've updated Prettier to its latest version (1.x -> 2.x). It will not cause any issues to your application, but may break snapshot tests. Your code will have to be reformatted with the latest Prettier version.
+
+It introduced trailing commas, parents for single param arrow functions and other changes. You can read [Prettier 2.0 "2020" blog post](https://prettier.io/blog/2020/03/21/2.0.0.html) for further details.
+
+#### ESLint and rules updated to 7.x
+
+ESLint (and related rules) have been updated to their latest version. Please ensure your code match the latest rules if your app uses Front-Commerce's default rules. `lint-staged` has also been updated from 9.x to 10.x and requires node 10.13+.
```

Comment:
Lint staged was never set up in the skeleton to begin with. Not sure what we should do regarding the documentation in this case. I think I wouldn't mention it.

Reply:
Sounds good to me, it is also not very important.